fre/yamltools/tests/test_combine_yamls.py

```diff
@@ -134,7 +134,7 @@ def test_combined_compileyaml_validatefail():
 
     # Validate against schema; should fail
     wrong_combined = Path(f"{COMP_OUT_DIR}/combined-am5-wrong_datatype.yaml")
-    schema_file = os.path.join(f"{IN_DIR}","compile_yamls","schema.json")
+    schema_file = os.path.join(Path(TEST_DIR).resolve().parents[1], "gfdl_msd_schemas", "FRE", "fre_make.json")
```

Comment:
see above

Reply:
I see what you mean, especially for tests. Might be better to have that path laid out explicitly